products/compute/api.yaml

```diff
@@ -1256,60 +1253,282 @@ objects:
       error: !ruby/object:Api::Async::Error
         path: 'error/errors'
         message: 'message'
-    properties:
-      - !ruby/object:Api::Type::String
-        name: 'name'
-        required: true
-        input: true
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'region'
+        resource: 'Region'
+        imports: 'name'
         description: |
-          Name of the resource. Provided by the client when the resource is
-          created. The name must be 1-63 characters long, and comply with
-          RFC1035. Specifically, the name must be 1-63 characters long and match
-          the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the
-          first character must be a lowercase letter, and all following
-          characters must be a dash, lowercase letter, or digit, except the last
-          character, which cannot be a dash.
-      - !ruby/object:Api::Type::Array
-        name: 'healthChecks'
-        item_type: Api::Type::String
+          A reference to the region where the regional backend service resides.
         required: true
-        min_size: 1
-        max_size: 1
+    properties:
+      - !ruby/object:Api::Type::Integer
+        name: 'affinityCookieTtlSec'
         description: |
-           The list of HealthChecks for checking the health of the backend service.
-           Currently at most one health check can be specified, and a health check
-           is required.
+          Lifetime of cookies in seconds if session_affinity is
+          GENERATED_COOKIE. If set to 0, the cookie is non-persistent and lasts
+          only until the end of the browser session (or equivalent). The
+          maximum allowed value for TTL is one day.
+
+          When the load balancing scheme is INTERNAL, this field is not used.
       - !ruby/object:Api::Type::Array
         name: 'backends'
         description: |
-          The list of backends that serve this RegionBackendService.
+          The set of backends that serve this RegionBackendService.
         item_type: !ruby/object:Api::Type::NestedObject
           properties:
+            - !ruby/object:Api::Type::Enum
+              name: 'balancingMode'
```

Comment:
Mind adding a test or 2 to GA that test BalancingMode and the other added fields?